src/TesApi.Web/AzureProxy.cs

```diff
@@ -478,6 +491,32 @@ public async Task DeleteBatchPoolIfExistsAsync(string poolId, CancellationToken
             }
         }
 
+        /// <inheritdoc/>
+        public Task<CloudPool> GetBatchPoolAsync(string poolId, DetailLevel detailLevel = default, CancellationToken cancellationToken = default)
+            => batchClient.PoolOperations.GetPoolAsync(poolId, detailLevel: detailLevel, cancellationToken: cancellationToken);
+
+        /// <inheritdoc/>
+        public Task CommitBatchPoolChangesAsync(CloudPool pool, CancellationToken cancellationToken = default)
+            => pool.CommitChangesAsync(cancellationToken: cancellationToken);
+
+        /// <inheritdoc/>
+        public async Task<(int? lowPriorityNodes, int? dedicatedNodes)> GetCurrentComputeNodesAsync(string poolId, CancellationToken cancellationToken = default)
```

Comment:
A type for the return value, seems appropriate.

Reply:
Same answer as the `GetComputeNodeAllocationStateAsync` question, above.